web/templates/inline_topic_link.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if is_empty_string_topic}}
```

Comment:
Renamed this to `inline_topic_link_label`, since it doesn't actually contain the link syntax itself.